common/app/views/fragments/nav/newHeaderMenu.scala.html

```diff
@@ -0,0 +1,64 @@
+@()(implicit request: RequestHeader)
+
+@import common.{ NewNavigation, TopLevelSection, LinkTo }
+
+@sectionList(topLevelSection: TopLevelSection) = {
+    <li class="main-navigation__item navigation-border">
+            <!-- TODO: Firefox -->
+        <details>
+            <summary class="main-navigation__item__button">
+                @fragments.inlineSvg("arrow-down", "icon")
+                @topLevelSection.name
+            </summary>
+            <ul class="main-navigation__secondary navigation-group">
+                @NewNavigation.sectionItems(topLevelSection).map { sectionItem =>
+                    <li class = "navigation-group__item" >
+                        <a href="@LinkTo { sectionItem.url }">@sectionItem.name</a>
+                    </li>
+                }
+            </ul>
+        </details>
+    </li>
+}
+
+<div class="main-menu-container" id="main-menu">
+        <!-- TODO: Provide tabbable alternative -->
+    <a href="#" class="main-menu-container__overlay"></a>
```

Comment:
scratch that. I see why 😃  That's elegant. Because it is a visual feature, I would then add the `aria-hidden="true"` attribute.